ports/ports.go

```diff
@@ -52,6 +54,14 @@ func PortToHostPort(port int) string {
 	return ":" + strconv.Itoa(port)
 }
 
+// HostPortToPort converts the  host:port address string intoto port int
+func HostPortToPort(hostPort string) (int, error) {
+	s := strings.Split(hostPort, ":")
```

Comment:
codecov is saying that those lines aren't covered by tests. Would you please add one test specific to this function?